.rubocop.yml

```diff
@@ -29,3 +29,6 @@ Metrics:
 
 Layout/HashAlignment:
   Enabled: false
+
+Gemspec/RequireMFA:
+  Enabled: false
```

Comment:
Not blocking this PR, but you probably want to enable MFA to publish to Rubygems.

Reply:
This is how the build team has it setup for automated release. Turning this on will result in failure. We had to add this exception for the main gem too unfortunately